src/deluge/drivers/uart/uart.h

```diff
@@ -32,7 +32,7 @@
  * and set HAVE_RTT to 0. Though this configuration hasn’t been tested for a while...
  */
 #ifndef ENABLE_TEXT_OUTPUT
-#define ENABLE_TEXT_OUTPUT 0
+#define ENABLE_TEXT_OUTPUT 1
```

Comment:
Is this intentional or did it slip in? The cmake config assumes the default is 0 and sets it to 1 in debug and relwithdebinfo builds

Reply:
We can drop this part too and let cmake handle matters.